main.cpp

```diff
@@ -10,48 +10,56 @@
 using namespace std;
 using namespace sf;
 
+struct USER
+{
+    string name;
+    string password;
+    bool successfullyCheck = 0;
+};
+
+struct GOODS
+{
+    vector <string> name;
+    vector <string> count;
+};
+
 string deleteSpaceInString (string name);
 static int login_callback(void *unused, int count, char **data, char **columns);
 static int listOfGoods_callback(void *unused, int count, char **data, char **columns);
 static int checkForNull_callback(void *unused, int count, char **data, char **columns);
-int showLogin(int width, int hight, int frame, Font font);
-void showShop (int width, int hight, int frame, Font font);
-
-string GlobalName = "";
-string GlobalPassword = "";
-
-bool loginSucceffuly = false;
+int showLogin();
+void showShop ();
+bool loginUser(string inputName, string inputPassword);
+GOODS createListOfGoods();
+void editCount(int button, GOODS goods);
+GOODS updateGoods();
+Font getFont();
 
-vector <string> prod; 
-vector <string> prodCount;
+const int WIDTH = 400;
+const int HIGHT = 600;
+const int FRAME = 11;
+const Font FONT = getFont();
+const char *WAYTODB = "src/Database.db";
 
-string GlobalCount;
 
 int main() {
-    int width = 400, hight = 600;
-    int frame = 11;
-    Font font;
-    font.loadFromFile("src/Mooli-Regular.ttf");
-    if (!font.loadFromFile("src/Mooli-Regular.ttf"))
-    {
-    cout << "Error to load font" << endl;
-    }
-    int CheckForCLose = showLogin(width, hight, frame, font);
-    if (CheckForCLose == 1){
+
+    if (showLogin() == 1){
         return 0;
     }
-    showShop(width, hight, frame, font);
+    showShop();
 
     return 0;
 }
 
-int showLogin(int width, int hight, int frame, Font font){
+int showLogin(){
     
-    RenderWindow window(VideoMode(width,hight), "main");
+    RenderWindow window(VideoMode(WIDTH,HIGHT), "login");
```

Comment:
Set space between comma and second param
```suggestion
    RenderWindow window(VideoMode(WIDTH, HIGHT), "login");
```